src/tools/common/sss_tools.h

```diff
@@ -92,7 +92,7 @@ errno_t sss_tool_popt_ex(struct sss_cmdline *cmdline,
                          const char *fopt_name,
                          const char *fopt_help,
                          enum sss_tool_opt fopt_require,
-                         const char **_fopt,
+                         char **_fopt,
```

Comment:
Do you need this change to be able to `free(*_fopt)`?

Reply:
Yes, if I don't do that then I get a warning about discarding `const` from the pointer.

There seems to be some discussion about it in [stackoverflow](https://stackoverflow.com/questions/2819535/unable-to-free-const-pointers-in-c).